pkg/report/output/output.go

```diff
@@ -22,99 +24,94 @@ import (
 
 var ErrUndefinedFormat = errors.New("undefined output format")
 
-func GetOutput(
+func GetData(
 	report globaltypes.Report,
 	config settings.Config,
 	baseBranchFindings *basebranchfindings.Findings,
-) (*types.Output[any], error) {
-	var err error
-	var output types.GenericOutput
+) (*types.ReportData, error) {
+	sendToCloud := false
+
+	data := &types.ReportData{}
+	// add detectors
+	err := detectors.AddReportData(data, report, config)
+	if config.Report.Report == flag.ReportDetectors {
```

Comment:
I think we'll miss errors for non-detectors reports:
```suggestion
	if config.Report.Report == flag.ReportDetectors || err != nil {
```